frontend/javascripts/admin/voxelytics/artifacts_disk_usage_list.tsx

```diff
@@ -0,0 +1,88 @@
+import React from "react";
+import { Table } from "antd";
+import _ from "lodash";
+
+import { VoxelyticsArtifactConfig, VoxelyticsTaskConfigWithHierarchy } from "types/api_flow_types";
+import { formatCountToDataAmountUnit } from "libs/format_utils";
+import { ColumnsType } from "antd/lib/table";
+import { renderArtifactPath } from "./artifacts_view";
+import { localeCompareBy } from "libs/utils";
+
+type ArtifactTableEntry = {
+  artifactName: string;
+  artifactSize: string;
+  taskName: string;
+  fileSize: number;
+  inodes: string;
+  inodeCount: number;
+  filePath: string;
+  filePathComponent: React.ReactNode;
+};
+
+export default function DiskUsageList({
+  tasksWithHierarchy,
+  artifacts,
+}: {
+  tasksWithHierarchy: Array<VoxelyticsTaskConfigWithHierarchy>;
+  artifacts: Record<string, Record<string, VoxelyticsArtifactConfig>>;
+}) {
+  function taskToTableEntry(
+    taskGroup: VoxelyticsTaskConfigWithHierarchy,
+  ): Array<ArtifactTableEntry> {
+    if (taskGroup.isMetaTask) {
+      return _.flatten(
+        taskGroup.subtasks
+          .map(taskToTableEntry)
+          .filter((artifactTableEntry) => artifactTableEntry != null),
+      );
+    }
+    return Object.entries(artifacts[taskGroup.taskName]).map(([artifactName, artifact]) => ({
+      artifactName,
+      artifactSize: formatCountToDataAmountUnit(artifact.fileSize),
+      fileSize: artifact.fileSize,
```

Comment:
isn't this the same? the first entry is formatted and the second is not?

Reply:
You are right: one is the formatted value and one is the unformatted raw value.
The unformatted raw value is used to be able to sort the artifact entries properly by their size.
The formatted value is simply the value that is displayed in the table.


I looked at the docs that list the `render` prop for the columns prop of a table. I updated the artifact entries to only have the file size and to format them in the render function.
This way is much better as the filesize is not contained in every entry twice. :+1: 

Thanks for mentioning this :pray:

Reply:
great :+1: did you push this already?

Reply:
Sorry, I got interrupted a little. It is pushed now :)